frontend/src/components/Transactions/TransactionLink.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="transaction-slot-link">
+    <div @click="visible = !visible">
+      <!-- Collaps Icon  -->
+      <collapse-icon class="text-right" :visible="visible" />
+      <div>
+        <b-row>
+          <!-- ICON  -->
+          <b-col cols="1">
+            <type-icon color="text-danger" icon="share" />
+          </b-col>
+
+          <b-col cols="11">
+            <!-- Amount / Name || Text -->
+            <amount-and-name-row :amount="amount" :text="$t('gdd_per_link.links_sum')" />
+
+            <!-- Count Links -->
+            <link-count-row count="22" />
+
+            <!-- Decay -->
+            <decay-row :decay="decay" />
```

Comment:
decay is not negative - clarify with @Mogge 

![image](https://user-images.githubusercontent.com/1238238/158023930-6fbe5c17-f8d5-465c-8f18-a4bead1d9d49.png)